pkg/plugins/policies/meshcircuitbreaker/api/v1alpha1/validator.go

```diff
@@ -170,7 +172,14 @@ func validateDetectorSuccessRate(path validators.PathBuilder, detector *Detector
 
 	verr.Add(validators.ValidateIntegerGreaterThanZeroOrNil(path.Field("minimumHosts"), detector.MinimumHosts))
 	verr.Add(validators.ValidateIntegerGreaterThanZeroOrNil(path.Field("requestVolume"), detector.RequestVolume))
-	verr.Add(validators.ValidateIntegerGreaterThanZeroOrNil(path.Field("standardDeviationFactor"), detector.StandardDeviationFactor))
+	if detector.StandardDeviationFactor != nil {
+		dec, err := common_api.NewDecimalFromIntOrString(*detector.StandardDeviationFactor)
```

Comment:
Can `ToDecimal` be a method on the IntOrString type?

Reply:
How could we implement this? `IntOrString` is Kubernetes type. I could not put our type alias to `IntOrString` in the API :(

Reply:
Ah ok there's no alias, nvm